<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoubleEmForm</class>
 <widget class="QWidget" name="DoubleEmForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(114, 112, 208);</string>
  </property>
  <widget class="QLabel" name="dp_1_label">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>194</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/wheel_run.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="vc_2_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>75</y>
     <width>13</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/vertical_connector_1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dp_2_label">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>193</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/wheel_stop.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="status_label">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>130</y>
     <width>62</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/switchstatus.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="status_label_2">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>129</y>
     <width>62</width>
     <height>64</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/switchstatus.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="coro_1_label">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>136</y>
     <width>17</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/coro.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="coro_text_1_label">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>121</y>
     <width>24</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>正向</string>
   </property>
  </widget>
  <widget class="QLabel" name="coro_text_2_label">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>121</y>
     <width>24</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>正向</string>
   </property>
  </widget>
  <widget class="QLabel" name="hc_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>75</y>
     <width>113</width>
     <height>11</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/horizon_pipe.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="inve_text_2_label">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>121</y>
     <width>24</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>反向</string>
   </property>
  </widget>
  <widget class="QLabel" name="em_label">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>6</y>
     <width>58</width>
     <height>58</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/em_run.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="vc_1_label">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>62</y>
     <width>13</width>
     <height>68</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/vertical_connector_1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="inve_text_1_label">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>121</y>
     <width>24</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>反向</string>
   </property>
  </widget>
  <widget class="QLabel" name="coro_2_label">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>135</y>
     <width>17</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/coro.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="inve_1_label">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>136</y>
     <width>17</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/inve.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="inve_2_label">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>135</y>
     <width>17</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/inve.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
